app/views/shared/_card-listings-l.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="card-listings-l">
+  <div class="picture">
+    <img src="#" alt="">
+    <div class="profile d-flex">
+      <%= image_tag "source", alt: "alttext" %>
+      <%= @user.name %>
+    </div>
+    <div class="skill-tag">
+    </div>
+  </div>
+  <div class="info">
+    <p class="location"><%=  %></p>
```

Comment:
Just making sure--these <%= %> below were intentionally left blank, right?

Reply:
it is intentionally blank! i cannot see anything at the moment so,,, just a blank.

Reply:
Awesome!